geoportailv3/static/js/draw/drawnfeaturesservice.js

```diff
@@ -1,26 +1,203 @@
 /**
- * @fileoverview Provides a selected features service useful to share
+ * @fileoverview Provides a dranw features service useful to share
  * information about the selected feature throughout the application.
```

Comment:
"selected"? Are you sure?

I think it's worth having descriptive and accurate `@fileoverview`'s.